logitrail-woocommerce.php

```diff
@@ -802,9 +817,9 @@ function logitrail_barcode_save($post_id){
 
     function enable_logitrail_shipping_save($post_id) {
         if( !empty($_POST['enable_logitrail_shipping']) ) {
-            update_post_meta( $post_id, 'enable_logitrail_shipping', esc_attr( $_POST['enable_logitrail_shipping'] ) );
+            update_post_meta( $post_id, 'logitrail_enable_shipping', esc_attr( $_POST['logitrail_enable_shipping'] ) );
```

Comment:
Some actual reason for this change? And now setting and function name don't match..